tools/environment-setup.sh

```diff
@@ -68,8 +71,8 @@ lxc exec snapcraft-dev -- sudo -iu ubuntu pip install \
 # Install the project for quick tests
 lxc exec snapcraft-dev -- sudo -iu ubuntu pip install --editable snapcraft
 
-# Install black to run static tests
-lxc exec snapcraft-dev -- snap install black --beta --devmode
+# Install black to run static tests, if available
```

Comment:
Mind dropping this now due to https://snapcraft.io/black being on all architectures (well, except i386)?

Reply:
done!